packages/grafana-ui/src/components/DateTimePickers/TimeRangeInput.tsx

```diff
@@ -31,6 +32,7 @@ export interface TimeRangeInputProps {
   /** Controls visibility of the preset time ranges (e.g. **Last 5 minutes**) in the picker menu */
   hideQuickRanges?: boolean;
   disabled?: boolean;
+  icon?: IconName;
```

Comment:
I'm wondering if we want to allow to pass a custom icon or if we just want to use the same one we use in other time pickers? They use "clock-nine" and then we could have a `showIcon` prop. Does that work for you and what do you think about it? Just seems strange to allow to pass in a custom icon to a time picker...

Reply:
make sense, will do the change